src/probabilistic.jl

```diff
@@ -58,13 +71,10 @@ function approximate_negative_log_evidence(f, θ, verbose::Bool = false)
 	# println("val: $(val)")
 	# println("rest: $(logdet(Σ) + d * log(2π))")
 	try
-		logdet(Σ)
+		return val - (logdet(Σ) + d * log(2π)) / 2
```

Comment:
The previous version computed `logdet` twice. Shouldn't have a huge impact because the matrices are mostly small.